packages/sdk/src/browser-wallets/unisat/constants.ts

```diff
@@ -0,0 +1,7 @@
+import type { Network } from "../../networks/types";
+
+export const NETWORK_TO_UNISAT_NETWORK: Record<Network, UnisatNetwork | ""> = {
+  mainnet: "livenet",
+  testnet: "testnet",
+  regtest: "", // Unsupported by Unisat
+} as const;
```

Comment:
The user can still enter `"regtest"` in `getAddresses`, this will skip the check for targetNetwork and leave handling to unisat itself. I don't think we should do this because we can simply throw an error that ordit-sdk doesn't support it right now, and keeping it here will help code maintenance by avoiding checking for `"regtest"` directly in the function .

Reply:
XVerse and Unisat both the wallets does not support regtest. Same for Coin98 and Leather wallet. My point is we can keep a separate network type for browser wallets and that keep the types to the point. I don't think it helps us pass regtest down the chain when its never going to be used. We can avoid empty strings by removing that.

> The user can still enter "regtest" in getAddresses

That's exactly why i have suggested `BrowserWalletNetworks` that would be replaced w/ `Network` below on L28: 

https://github.com/ordzaar/ordit-sdk/blob/baa0e58efa0df4c5c0148e49d3efd669ef7b7c62/packages/sdk/src/browser-wallets/unisat/index.ts#L27-L29

Reply:
Removed! Resolving this thread.